ibm/resource_ibm_private_dns_custom_resolver_location.go

```diff
@@ -95,6 +97,17 @@ func resourceIBMPrivateDNSLocationCreate(context context.Context, d *schema.Reso
 		return diag.FromErr(fmt.Errorf("Error creating the custom resolver location %s:%s", err, resp))
 	}
 	d.SetId(convertCisToTfThreeVar(*result.ID, resolverID, instanceID))
+	_, err = waitForPDNSCustomResolverHealthy(d, meta)
+	if err != nil {
+		return diag.FromErr(err)
+	}
+	optCr := sess.NewUpdateCustomResolverOptions(instanceID, resolverID)
+	optCr.SetEnabled(true)
+	resultCr, respCr, errCr := sess.UpdateCustomResolverWithContext(context, optCr)
```

Comment:
it should be enabled only if user sets enabled to true.. we cannot hardcode it..

Reply:
Added check to see if enable is set before enabling